python/pytest/test_tests.py

```diff
@@ -29,36 +29,53 @@ def test_bug1(device):
 
 
 
-# class TestGather(object):
-#     h=1e-3
-    
-#     def backprop(self,cls, N,nc, device):
-#         atoms=ptens_base.atomspack.random(N, nc, 0.3)
-#         x=cls.randn(atoms,nc).to(device)
-#         x.requires_grad_()
-#         G=ptens.ggraph.random(N,0.3)
-#         atoms2 = G.subgraphs(ptens.subgraph.trivial())
-
-#         check = gradcheck(cls.gather, (atoms2, x), eps=self.h)
-#         assert check
-
-#         z = cls.gather(atoms2, x)
-#         loss=torch.sum(z)
-#         loss.backward()
-#         xgrad=x.grad
-
-
-#         fn = lambda x: cls.gather(atoms2, x)
-#         xgrad2 = numerical_grad_sum(fn, x, self.h)
-        
-#         assert torch.allclose(xgrad, xgrad2, rtol=1e-2, atol=1e-2)
+graph_atoms_list= [
+    (ptens.ggraph.from_edge_index( torch.Tensor([[0, 1], [1, 0]]).int() ),
```

Comment:
@risi-kondor here is where I create the Graph and Atomspack that form the ptensorlayer0 for the test.

In the previous test both were random.
But I realized that this one already creates the problem.